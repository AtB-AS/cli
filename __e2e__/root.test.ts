import {
  spawnScript,
  getTempDirectory,
  cleanup,
  writeFiles,
} from '../jest/helpers';

const cwd = getTempDirectory('test_different_roots');

beforeEach(() => {
  cleanup(cwd);
  writeFiles(cwd, {});
});
afterEach(() => {
  cleanup(cwd);
});

test('works when Gradle is run outside of the project hierarchy', () => {
  const {stdout} = spawnScript('echo', ['hello world'], {cwd});
  expect(stdout).toMatchSnapshot();
});
