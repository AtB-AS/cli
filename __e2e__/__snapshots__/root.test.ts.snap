// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works when Gradle is run outside of the project hierarchy 1`] = `"hello world"`;
